fixtures/tmb/solaris-flare.json

```diff
@@ -0,0 +1,850 @@
+{
+  "$schema": "https://raw.githubusercontent.com/FloEdelmann/open-fixture-library/master/schemas/fixture.json",
+  "name": "Solaris Flare",
+  "shortName": "TMBFlare",
+  "categories": ["Strobe", "Blinder", "Color Changer"],
+  "meta": {
+    "authors": ["Flo Edelmann"],
+    "createDate": "2018-03-15",
+    "lastModifyDate": "2018-03-15"
+  },
+  "comment": "Flares with serial numbers that contain “14R” have RDM functionality.\nExample: xxxxxx-14R-xxxx",
+  "manualURL": "http://pub.tmb.com/solaris/pdf/Solaris-Flare-Manual.pdf",
+  "rdm": {
+    "modelId": 1703,
+    "softwareVersion": "9.3C"
+  },
+  "physical": {
+    "dimensions": [210, 232, 497],
+    "weight": 9.5,
+    "power": 1000,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "96 10W RGBW Cree LEDs"
+    },
+    "lens": {
+      "degreesMinMax": [36, 36]
+    },
+    "focus": {
+      "type": "Fixed"
+    }
+  },
+  "matrix": {
+    "pixelCount": [16, 6, 1],
+    "pixelGroups": {
+      "1 Pixel": [
+        "(1, 1)", "(2, 1)", "(3, 1)", "(4, 1)", "(5, 1)", "(6, 1)", "(7, 1)", "(8, 1)", "(9, 1)", "(10, 1)", "(11, 1)", "(12, 1)", "(13, 1)", "(14, 1)", "(15, 1)", "(16, 1)",
+        "(1, 2)", "(2, 2)", "(3, 2)", "(4, 2)", "(5, 2)", "(6, 2)", "(7, 2)", "(8, 2)", "(9, 2)", "(10, 2)", "(11, 2)", "(12, 2)", "(13, 2)", "(14, 2)", "(15, 2)", "(16, 2)",
+        "(1, 3)", "(2, 3)", "(3, 3)", "(4, 3)", "(5, 3)", "(6, 3)", "(7, 3)", "(8, 3)", "(9, 3)", "(10, 3)", "(11, 3)", "(12, 3)", "(13, 3)", "(14, 3)", "(15, 3)", "(16, 3)",
+        "(1, 4)", "(2, 4)", "(3, 4)", "(4, 4)", "(5, 4)", "(6, 4)", "(7, 4)", "(8, 4)", "(9, 4)", "(10, 4)", "(11, 4)", "(12, 4)", "(13, 4)", "(14, 4)", "(15, 4)", "(16, 4)",
+        "(1, 5)", "(2, 5)", "(3, 5)", "(4, 5)", "(5, 5)", "(6, 5)", "(7, 5)", "(8, 5)", "(9, 5)", "(10, 5)", "(11, 5)", "(12, 5)", "(13, 5)", "(14, 5)", "(15, 5)", "(16, 5)",
+        "(1, 6)", "(2, 6)", "(3, 6)", "(4, 6)", "(5, 6)", "(6, 6)", "(7, 6)", "(8, 6)", "(9, 6)", "(10, 6)", "(11, 6)", "(12, 6)", "(13, 6)", "(14, 6)", "(15, 6)", "(16, 6)"
+      ],
+      "2 Pixel V 1": [
```

Comment:
I`d prefer the format `2V-Pixel 1". Please use a hyphen between pixel number and "Pixel" and put "2V" together (as well as "2H", "6V", ...) as it's the same in the fixture's menu.